app/va/vetext/client.py

```diff
@@ -38,6 +38,8 @@ def send_push_notification(self, mobile_app: str, template_id: str, icn: str, pe
         }
         self.logger.info("VEText Payload information: %s", payload)
 
+        # monotonic() returns 0.0 so I explicitly set it 0.0
+        start_time = 0.0
         try:
             start_time = monotonic()
```

Comment:
```suggestion
        start_time = monotonic()
        try:
```
The monotonic call belongs outside the try block; that's not why the code is in a try block.